dataprofiler/data_readers/avro_data.py

```diff
@@ -47,13 +47,14 @@ def _load_data_from_file(self, input_file_path):
             lines = list()
             for line in df_reader:
                 lines.append(line)
-            df, original_df_dtypes = data_utils.json_to_dataframe(
-                    json_lines=lines,
-                    selected_columns=self.selected_keys,
-                    read_in_string=True
-            )
-            self._original_df_dtypes = original_df_dtypes
-            return df
+            return lines
+            #df, original_df_dtypes = data_utils.json_to_dataframe(
+            #        json_lines=lines,
+            #        selected_columns=self.selected_keys,
+            #        read_in_string=True
+            #)
+            #self._original_df_dtypes = original_df_dtypes
+            #return df
```

Comment:
remove? Also doesn't this potentially create issues because the types aren't discovered?